src/Endpoints/Users.php

```diff
@@ -13,18 +14,21 @@
  *
  * @package BestIt\Harvest\Endpoints
  * @author Ahmad El-Bardan <ahmad.el-bardan@bestit-online.de>
+ * @author Marcel Thiesies <marcel.thiesies@bestit-online.de>
  */
 class Users extends BaseEndpoint
 {
     /**
      * Retrieve all users.
      *
+     * @param DateTime|null $updatedSince
      * @return UsersModel
-     * @TODO: filters
      */
-    public function all()
+    public function all(DateTime $updatedSince = null)
     {
-        $response = $this->httpClient->get('/people');
+        $uri = '/people';
```

Comment:
This does not need to be an extra variable. It can be inlined in the call to `Utils:: appendUpdatedSinceToUri`.